CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Extensions/ReuseIdentifier.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ReuseIdentifier.swift
+//  
+//
+//  Created by David Vizaknai on 13.09.2022.
+//
+
+import UIKit
+
+public protocol Reusable {
+    static var reuseIdentifier: String { get }
+}
+
+public extension Reusable {
+    static var reuseIdentifier: String {
+        String(describing: self)
+    }
+}
+
+extension UICollectionReusableView: Reusable {}
```

Comment:
[Suggestion] I think, it is better to have explicit protocol conformance for this protocol and not to generalise it because at the moment most of cells don’t use it.